Drivers/Src/LTC681x.c

```diff
@@ -273,7 +272,7 @@ void LTC681x_adcvax(
   md_bits = (MD & 0x02) >> 1;
   cmd[0] = md_bits | 0x04;
   md_bits = (MD & 0x01) << 7;
-  cmd[1] =  md_bits | ((DCP&0x01)<<4) + 0x6F;
+  cmd[1] =  md_bits | ((DCP&0x01)<<4) | 0x6F;
```

Comment:
Why was this changed?

Reply:
This was giving a compiler warning, I tested it to make sure nothing changed